specification/components/schemas/eRS-Appointment.yaml

```diff
@@ -70,6 +132,7 @@ properties:
     maxItems: 1
   created:
     type: string
+    nullable: true
```

Comment:
I think an Appointment should always have a created date time, do we sometimes not include this field?

Reply:
Yes, it not included (or checked) on book flow as it gets generated on the server side. We actually send it as null from our regression pack.